src/reducers/versions.tsx

```diff
@@ -586,7 +586,7 @@ export const getDiffAnchors = (diff: DiffInfo): string[] => {
 };
 
 export type GetRelativeDiffAnchorParams = {
-  currentAnchor?: string | void;
+  currentAnchor?: string;
```

Comment:
For consistency with the other `GetRelative*` definitions, could you make this `string | undefined`? It's saying that that the `currentAnchor` property can be omitted or it can be explicitly set to `undefined`. I see that TS doesn't complain either way but keeping the calls consistent might avoid future surprises.

Reply:
I updated it.